_includes/2018-hero.html

```diff
@@ -9,16 +9,16 @@ <h3 style="margin-bottom: 1.3em; margin-top: 0;">Toronto, Ontario</h3>
   <!-- Event hero section -->
   <section>
     <div class="feature">
-      <a class="splash-link" href="/yami-ichi/"><h4 class="splash-heading">Kickoff Yami-ichi {% include graphics/bullet.svg %} 13</h4></a>
-      <p>Network field trips and Toronto's first internet yami-ichi</p>
+      <a class="splash-link" href="/yami-ichi/"><h4 class="splash-heading">Internet Yami-ichi {% include graphics/bullet.svg %} 13</h4></a>
+      <p>Kickoff with network field trip and Toronto's first Yami-ichi</p>
     </div>
     <div class="feature">
       <a class="splash-link" href="/conference/"><h4 class="splash-heading">Conference {% include graphics/bullet.svg %} 14, 15</h4></a>
-      <p>Our main event, a weekend full of talks, demos, and workshops</p>
+      <p>Our main event: a weekend full of talks, demos, and workshops</p>
```

Comment:
This seems like an outlier with the colon. I wonder if this could be something like: 
`Our conference weekend full of talks, demos, and weekends`?

Reply:
I think we can just trim the front? `A weekend full of talks, demos, and workshops` ? That is what I've pushed @garrying !